src/main/java/meditracker/storage/FilePathChecker.java

```diff
@@ -151,18 +151,17 @@ public static boolean isValidFullPath(Path path) {
             MEDILOGGER.warning("File does not end in .json");
             return false;
         }
-
         return true;
     }
 
     /**
-     * Validates the path input and return the Path object if successfully validated.
+     * Validates the string input and return the Path object if successfully validated.
      *
-     * @param fileLocationArgument The argument specifying the location of the file (as String)
-     * @return The Path object corresponding to the argument for the save file if it passes validation checks.
+     * @param fileLocationArgument The argument specifying the location of the file (as String).
+     * @return The Path object corresponding to the string argument for the save file if it passes validation checks,
```

Comment:
Should be fullstop, not comma

Reply:
there is further text "null otherwise" after the comma actually which is not modified (L162)

Reply:
Sry my bad.